assets/js/_main.js

```diff
@@ -32,7 +32,9 @@ $(document).ready(function(){
   });
 
   // init smooth scroll
-  $("a").smoothScroll({offset: -20});
+  $("a").smoothScroll({offset: -20, preventDefault:false});
+
+  anchors.add();
```

Comment:
If we can add "doc" class to the headings in docs somehow, then:
```js
anchors.add('h2.doc, h3.doc, h4.doc, h5.doc, h6.doc')
```
It uses document.querySelectorAll if you pass a string: https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll